package.json

```diff
@@ -17,7 +17,9 @@
     "eslint": "^5.11.0",
     "eslint-plugin-cypress": "^2.2.1",
     "http-server": "^0.11.1",
+    "jszip": "^3.5.0",
     "npx": "^10.2.0",
+    "recursive-readdir": "^2.2.2",
```

Comment:
research and tries give me convincing evidence recursivly walking a directory can't be easily achieved with node's fs module.